test/integration/targets/plugin_filtering/filter_modules.yml

```diff
@@ -0,0 +1,9 @@
+---
+filter_version: 1.0
+module_blacklist:
+  # A pure action plugin
+  - pause
+  # A hybrid action plugin with module
+  - copy
+  # A pure module
+  - tempfile
```

Comment:
Since the plugin filter should only applies to modules at the moment adding the name of a non-module plugin here could help verify it is module-only. Maybe blacklist a lookup plugin and then add a task that uses it in the intg tests to verify it wasnt blocked?

Reply:
Adding a win_* module or two here and testing may also potentially catch future problems.

Reply:
@alikins good idea.  I'll add something for that.